mp/src/game/client/momentum/clientmode_mom_normal.cpp

```diff
@@ -167,11 +162,11 @@ int ClientModeMOMNormal::HudElementKeyInput(int down, ButtonCode_t keynum, const
     }
 
     // Detach the mouse if the user right-clicked while the leaderboards are open
-    if (m_pLeaderboards && m_pLeaderboards->IsVisible())
+    if (g_pClientTimes && g_pClientTimes->IsVisible())
```

Comment:
`g_pClientTimes` should never be null here.

```suggestion
    if (g_pClientTimes->IsVisible())
```